### PR TITLE
BD-2837 Add hidden Link Shortening page and updated post/send

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-2837**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**Early Access launches 2/19/24**)
- [ ] No

### CODE REVIEW

**Comment by ianabels** [comment]
Two additional updates @rachel-feinberg 

- We can update the length of basic static URLs to 20 characters (from 20-21)
- Link expiration for personalized URLs is 2 months (from 3 months)

File: _docs/_hidden/private_betas/link_shortening.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+---
+nav_title: "Link Shortening"
+permalink: "/updated_link_shortening/"
+hidden: true
+---
+
+# Link Shortening
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/link_shortening.md:7]:
```suggestion
# Link shortening
```

File: _docs/_hidden/private_betas/link_shortening.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+For Braze to recognize URLs, they must start with _http://_ or _https://_. When a URL is recognized, the **Preview** pane will update with a placeholder URL. Braze will estimate the length of the URL after shortening, but a warning will prompt you to select a test user and save the message as a draft for a more accurate estimate.
+
+![][3]
+
+### Liquid personalization in URLs
+
+You can dynamically construct your URL directly within the Braze composer, allowing you to add dynamic UTM parameters to your URLs or send user’s unique links (such as directing users to their abandoned cart or to a specific product that is back in stock).
+
+#### Construct a URL with supported Liquid personalization tags
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/link_shortening.md:45]:
```suggestion
#### Create a URL with supported Liquid personalization tags
```
Alternative to the word "construct"? Feel free to disregard

File: _docs/_hidden/private_betas/link_shortening.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+{% raw %}
+```liquid
+https://example.com/?campaign_utm={{campaign.${api_id}}}&user_attribute={{custom_attribute.${attribute1}}}
+```
+{% endraw %}
+
+We also support the shortening of custom-defined Liquid variables. Several examples are shown below:
+
+#### Construct a URL using Liquid variables
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/link_shortening.md:57]:
```suggestion
#### Create a URL using Liquid variables
```

File: _docs/_hidden/private_betas/link_shortening.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+Dynamic URs are valid for two months from the time of URL registration.
+
+#### Will Link Shortening work with deep links or universal links?
+
+Link shortening will shorten any static URLs that start with _http://_ or _https://_. However, it is not advised to further shorten generated universal links (from providers such as Branch or Appsflyer) as this may break the attribution or redirect of those tools.
+
+### Custom domains
+
+#### Can delegated domains be shared across multiple subscription groups?
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/link_shortening.md:169]:
```suggestion
#### Can delegated domains be shared across multiple subscription groups?

```

File: _docs/_hidden/private_betas/link_shortening.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+Yes, a single domain can be used with multiple subscription groups. To do so, select the domain for each subscription group that it should be associated with.
+
+#### Can delegated domains be shared across multiple workspaces?
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/link_shortening.md:172]:
```suggestion
#### Can delegated domains be shared across multiple workspaces?

```

File: _docs/_hidden/private_betas/link_shortening.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+
+### Link shortening
+
+#### Are the links I receive when test sending real URLs?
+
+If the campaign has been saved as a draft before test sending, yes. Otherwise, it is a placeholder link. Note that the exact URL sent in a launched campaign may differ from the one sent via a test send.
+
+#### Does the Braze SDK need to be installed in order to shorten links?
+
+No, Link Shortening will work without any SDK integration.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/link_shortening.md:147]:
```suggestion
No. Link shortening works without any SDK integration.
```

**Review by lydia-xie — APPROVED** [review]
left a few comments/suggestions, but lgtm!

**Pull request merged**